tools/test-proxy/documentation/asset-sync/README.md

```diff
@@ -103,6 +105,30 @@ After assets have been restored and then modified (re-recorded etc.) a push will
 test-proxy push --assets-json-path <assetsJsonPath>
 ```
 
+#### Config Commands
+
+When a client provides the additional body key `x-recording-assets-file` to `/Record/Start` or `/Playback/Start`, the test-proxy will invoke that test using **external assets**.
+
+It's great that recordings are externalized, but this adds some complexity as these recordings don't live directly next to their test code anymore. The test-proxy provides the `config` verb to offer easy insight into interactions with the `assets.json` to assist with these complexities.
+
+Currently there are two sub-verbs for `test-proxy config`:
+
+- `locate`
+- `show`
+
+`locate`: Dumps which folder under the `.assets` folder contains your recordings. [See `config` usage in layout section below.](#layout-within-a-language-repo)
+`show`: Dumps the contents of the targeted assets.json.
+
+```bash
+# from C:/repo/azure-sdk-for-python, the root of the python language repository
+test-proxy config locate --assets-json-file sdk/keyvault/azure-keyvault-keys/assets.json
```

Comment:
Remind me, does this work without `--assets-json-file` if you're in a directory with an `assets.json` file?

Reply:
No currently.